src/dumpContent.ts

```diff
@@ -0,0 +1,112 @@
+#!npx vite-node
+import fs from 'fs'
+import { promisify } from 'util'
+
+type ObjData = { _id: string }
+type SearchData = { continuation?: string; objs: ObjData[] }
+type BlobsData = { private_access: { get: { url: string } } }
+
+const DIRECTORY = 'contentDump'
+
+const API_KEY = process.env.SCRIVITO_API_KEY
+const INSTANCE_ID = process.env.SCRIVITO_TENANT || ''
+
+if (INSTANCE_ID && API_KEY) {
+  clearDump()
+  await dumpContent()
+  console.log('\n✅ Dump complete.')
+} else {
+  console.error(
+    'Please provide SCRIVITO_TENANT and SCRIVITO_API_KEY of the instance to dump.',
+  )
+}
+
+function clearDump() {
+  fs.rmSync(DIRECTORY, { force: true, recursive: true })
+  fs.mkdirSync(`${DIRECTORY}/json`, { recursive: true })
+  fs.mkdirSync(`${DIRECTORY}/binary`, { recursive: true })
+}
+
+async function dumpContent() {
+  let continuation
+
+  do {
+    const data: SearchData = await fetchJson<SearchData>(
+      `workspaces/published/objs/search`,
+      JSON.stringify({
+        continuation,
+        include_objs: true,
+        options: { site_aware: true },
+        size: 10,
```

Comment:
Given the speed of this script is not 100% important: we could use `size: 1` to fetch one obj at a time. With that the printed indicator `.` would match the number of objs.

This would also allow to (maybe later) add a progress bar.

Reply:
Sounds good. Let's keep it like it is.